File: server/types/answer.go
package types

import "time"

// Answer ...
type Answer struct {
	ID         int       `json:"id"`
	Text       string    `json:"text"`
	QuestionID int       `json:"questionId"`
	CreatedAt  time.Time `json:"createdAt"`
	UpdatedAt  time.Time `json:"updatedAt"`
}

File: front/src/types/answer.ts
export interface IAnswer {
  id?: number;
  questionId: string;
  text: string;
  createdAt?: string;
  updatedAt?: string;
}

export interface IGetAnswer {
  questionId: string;
}

export interface IOneAnswer {
  answer?: IAnswer;
  loading: boolean;
  error?: string;
}
